lib/open_api_spex/server.ex

```diff
@@ -25,16 +25,11 @@ defmodule OpenApiSpex.Server do
   @doc """
   Builds a Server from a phoenix Endpoint module
   """
-  @spec from_endpoint(module, [otp_app: atom]) :: t
-  def from_endpoint(endpoint, opts) do
-    app = opts[:otp_app]
-    url_config = Application.get_env(app, endpoint, []) |> Keyword.get(:url, [])
-    scheme = Keyword.get(url_config, :scheme, "http")
-    host = Keyword.get(url_config, :host, "localhost")
-    port = Keyword.get(url_config, :port, "80")
-    path = Keyword.get(url_config, :path, "/")
+  @spec from_endpoint(module) :: t
```

Comment:
Can we leave `from_endpoint/2` for compatibility, perhaps with an `@deprecated` attribute?

Reply:
If you need this in order to safely release a minor version bump I have no objections.  I will update the patch later this morning.